crates/signer-web3/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "alloy-signer-web3"
+description = "Ethereum Web3 signer"
+
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = [
+    "-Zunstable-options",
+    "--generate-link-to-definition",
+    "--show-type-layout",
+]
+
+[lints]
+workspace = true
+
+[dependencies]
+alloy-consensus = { workspace = true, features = ["std"] }
+alloy-network.workspace = true
+alloy-primitives.workspace = true
+alloy-signer.workspace = true
+alloy-provider.workspace = true
+
+async-trait.workspace = true
```

Comment:
this doesn't need any additional deps, do we really need another crate for this?